hash.c

```diff
@@ -1902,16 +1902,14 @@ rb_check_hash_type(VALUE hash)
 
 /*
  *  call-seq:
- *    Hash.try_convert(obj) -> obj, new_hash, or nil
+ *    Hash.try_convert(object) -> object, new_hash, or nil
  *
- *  If +obj+ is a +Hash+ object, returns +obj+.
+ *  If +object+ is a hash, returns +object+.
  *
- *  Otherwise if +obj+ responds to <tt>:to_hash</tt>,
- *  calls <tt>obj.to_hash</tt> and returns the result.
+ *  Otherwise if +object+ does not respond to +:to_hash+, returns +nil+.
```

Comment:
Actually, I think I prefer the original. Having the negation first is more difficult to understand.

Reply:
Restored.

Reply:
`+object+` and `+obj+` are mixed now.

Reply:
And the case `object` does not respond to `to_hash` disappears.

Reply:
Thanks, @nobu;  this was careless of me.  Now fixed (I hope).